ctapipe/utils/CutFlow.py

```diff
@@ -25,6 +26,11 @@ def __init__(self, name="CutFlow"):
         """
         self.cuts = OrderedDict()
         self.name = name
+        warn(
```

Comment:
I think we talked about deprecation cycles in ctapipe somewhere and that we would do them until the first 1.0 release.

Reply:
cutflow is currently used heavily in Protopipe, so I will keep it around just that code is migrated. I guess that will be quick